assets/app/views/browse/_pod-details.html

```diff
@@ -17,8 +17,15 @@
         <dd>{{pod.status.podIP || 'unknown'}}</dd>
         <dt>Node:</dt>
         <dd>{{pod.spec.nodeName || 'unknown'}} <span ng-if="pod.status.hostIP && pod.spec.nodeName != pod.status.hostIP">({{pod.status.hostIP}})</span></dd>
-        <dt>Restart policy:</dt>
+        <dt>Restart Policy:</dt>
         <dd>{{pod.spec.restartPolicy || 'Always'}}</dd>
+        <dt ng-if-start="pod.spec.activeDeadlineSeconds">Active Deadline:</dt>
+        <dd ng-if-end>
+          {{pod.spec.activeDeadlineSeconds | humanizeDurationValue : 'seconds'}}
+          <span ng-if="pod.status.phase === 'Running'" class="text-muted">
+            (<duration-until-now timestamp="pod.status.startTime"></duration-until-now> elapsed)
```

Comment:
are we sure startTime will always be there? maybe the ng-if above should be `pod.status.phase === 'Running' && pod.status.startTime` ?